pages/about/AboutContextHandler.inc.php

```diff
@@ -79,6 +79,36 @@ function submissions($args, $request) {
 
 		$templateMgr->assign( 'submissionChecklist', $context->getLocalizedSetting('submissionChecklist') );
 
+		// Get section options for this context
+		$roleDao = DAORegistry::getDAO('RoleDAO');
+		$user = $request->getUser();
+
+		$canSubmitAll = false;
+		if ($user) {
+			$canSubmitAll = $roleDao->userHasRole($context->getId(), $user->getId(), ROLE_ID_MANAGER) ||
+				$roleDao->userHasRole($context->getId(), $user->getId(), ROLE_ID_SUB_EDITOR);
+		}
+
+		$sectionDao = Application::getSectionDAO();
+		$sections = $sectionDao->getTitles($context->getId(), !$canSubmitAll);
+
+		if (count($sections) > 0) {
+			array_walk($sections, function (&$item, $sectionId) use ($sectionDao) {
+				$item = array('title' => $item);
+
+				$section = $sectionDao->getById($sectionId);
+
+				$sectionPolicy = $section ? $section->getLocalizedPolicy() : null;
+				$sectionPolicyPlainText = trim(PKPString::html2text($sectionPolicy));
+				if (strlen($sectionPolicyPlainText) > 0)
+					$item['policy'] = $sectionPolicy;
+			});
+		} else {
+			AppLocale::requireComponents(LOCALE_COMPONENT_APP_AUTHOR); // for author.submit.notAccepting
```

Comment:
This locale key doesn't exist in OMP (which also uses this library). It would need to be added there too (with "press" instead of "journal").

Reply:
Should i provide a PR for OMP also? Or is there another possible solution?

Reply:
Please! That's normally what we do. The codebases are pretty symmetrical (and getting more so as we further align them).

Reply:
Done via https://github.com/pkp/omp/pull/602/commits/1d8399c28001c2900918b1ce05c6f316c299fbd1